### ISSUE TITLE
build is failing

### ISSUE THREAD

**Issue by 7yl4r** [marinebon/old_marinebon.github.io#16]
Builds here seem to be having [this issue](https://github.com/actions/checkout/issues/760) but the workarounds suggested there do not seem to be working. I looked into the source code of the jekyll-build action and was not able to determine where that `docker run` command is being generated.

**Comment by torieketcham** [comment]
@7yl4r Is this the same issue that I am having?
<img width="1329" alt="Screen Shot 2022-07-20 at 11 47 48 AM" src="https://user-images.githubusercontent.com/44508950/180026480-b8a19822-f459-41c4-8eec-065d89f86a03.png">

**Comment by 7yl4r** [comment]
I suspect so. I have just been doing manual builds since we are expecting to move over to the new site soon.

**Comment by torieketcham** [comment]
Okay gotcha. Okay if I ask you to do another build later today?

**Comment by 7yl4r** [comment]
Yes. Just let me know when and I'll push the button on my computer.

**Comment by torieketcham** [comment]
Okay, I think it's ready to go. I can't guarantee that I won't have additional tweaks but I'm trying to keep them to a minimum. Can you do a build when you have a chance?

**Comment by 7yl4r** [comment]
OK. Done. I checked the site and I think it looks good.

**Comment by torieketcham** [comment]
Thanks Tylar. I had to make a couple of small tweaks (I named one of the files 2002 instead of 2022, eek) - could you re-run when you have a moment?

**Comment by 7yl4r** [comment]
done!

**Comment by torieketcham** [comment]
Thanks!

**Comment by torieketcham** [comment]
Ugh sorry - can I ask you to rebuild one more time? No rush.

**Comment by 7yl4r** [comment]
Just built & pushed.
